src/app/games/tafl/TaflState.ts

```diff
@@ -37,4 +38,13 @@ export class TaflState extends GameStateWithTable<TaflPawn> {
         return this.getHeight();
     }
 
+    public hasAbsoluteOwnerAt(coord: Coord, owner: PlayerOrNone): boolean {
```

Comment:
wtf is an absolute owner, why not simply owner?

Reply:
oui j'avais faire absoluteOwner/relativeOwner pour être explicite
mais owner/relativeOwner c'est bien aussi (c'est PlayerOrNone vs RelativePlayer after all)